src/monit/patch/0002-change_monit_alert_log_error.patch

```diff
@@ -0,0 +1,71 @@
+From 97a5defc6a7fcc6a00f691bb5314ceb8fb7704e9 Mon Sep 17 00:00:00 2001
+From: Abhishek Dosi <abdosi@microsoft.com>
+Date: Mon, 26 Oct 2020 11:40:02 -0700
+Subject: [PATCH] Patch on top of commit Patch is addressing these changes:-
+
+a) Enable repeat keyword for alert action . This was we can log
+syslog error message for continuous failure condition.
+
+b) Make sure log and error message are different so that we do not rate
+limit error message with log message
+
+c) Make sure error message is loggged is state change to fail first time
```

Comment:
Phrasing is a bit confusing, e.g., "is loggged is"

Reply:
@jleveque Updated the comments with more better description. Ans updated order of change based on priority.